src/app.js

```diff
@@ -300,19 +308,38 @@ app.routeAsync("/groups/:groupName")
   .getAsync(endpoints.static.sendGatsbyEntrypoint)
 ;
```

Comment:
I'm still not sure the authz layers are quite right in this latest pass. For example, this route isn't protected any more and it should be.

Reply:
Good catch here, that wasn't intentional. I dropped 0af95cc316c56f19f94b553ff72110f7b4532273 which should fix things since:

1. Without it, the authz assertion from `setSource` is retained.
2. Having the Group in the request's context is only necessary for the new `settings` endpoints, and that's already set from 833ea30fa2b5b97c931a9b8d94a4560b255f34ae.

Reply:
> Does 7184cd7295ac006cfb6f5fa981c1d4b6146ea2f4 still apply?

Oh right it doesn't after the change above. Also dropped.